hail/src/main/scala/is/hail/types/physical/PCanonicalNDArray.scala

```diff
@@ -126,6 +126,10 @@ final case class PCanonicalNDArray(elementType: PType, nDims: Int, required: Boo
     ))
   }
 
+  def mutateElement(indices: IndexedSeq[Value[Long]], ndAddress: Value[Long], newElement: Code[_], mb: EmitMethodBuilder[_]): Code[Unit] = {
```

Comment:
I think I'd expect this to be called `setElement`. What do you think?

Reply:
Done